### PR TITLE
PokeApi JS Wrapper Caching

### LINKED ISSUE #29: PokeAPI JS Wrapper Configuration
## Expected Behavior

The PokeAPI wrapper constructor should be called with an `options` parameter.  (In your app, it is called `interval` in ApiCall.js on line 4)

## Actual/Current Behavior

The PokeAPI wrapper constructor is not being passed a configuration object, ex: `const P = new Pokedex.Pokedex(interval);`

## Learning

https://github.com/PokeAPI/pokeapi-js-wrapper#configuration

### PR DESCRIPTION
**Pull request by krferrell**
## Changes
1. Implemented caching properly

## Purpose
The service worker was not properly implemented. 

## Approach
Pass a custom option object to the right function.

Closes #29

### CODE REVIEW

File: pokeapp/src/utilities/ApiCall.js [hunk 1]
```diff
@@ -1,12 +1,17 @@
...
   limit: 1119,
-  cache: true,
 };
 
+const customOptions = {
+  cache: true,
+}
+
+
+const P = new Pokedex.Pokedex(customOptions);
```
Comment (BlackCubes) [inline pokeapp/src/utilities/ApiCall.js:13]: It's just me, but maybe give a better descriptive variable? But, if it works then it works 😘

**Review by BlackCubes — APPROVED** [review]
Looks gewd!

**Review by dannysanchez559 — APPROVED** [review]
Any warnings in the browser with this method?

File: pokeapp/src/utilities/ApiCall.js (hunk 1 above)
Comment (krferrell) [reply pokeapp/src/utilities/ApiCall.js:13]: Thanks for the input!

File: pokeapp/src/utilities/ApiCall.js [hunk 2]
```diff
@@ -1,12 +1,17 @@
...
 
 const interval = {
   offset: 0,
   limit: 1119,
-  cache: true,
 };
 
+const customOptions = {
+  cache: true,
+}
```
Comment (kmausolf) [inline pokeapp/src/utilities/ApiCall.js:10]: This should include more than just a cache flag.  Please see this doc for more info:  https://github.com/PokeAPI/pokeapi-js-wrapper#configuration

**Pull request merged**